requirements/CI/requirements.txt

```diff
@@ -8,8 +8,8 @@ sphinx-argparse==0.2.5
 sphinx-issues==1.2.0
 sphinx_rtd_theme==0.5.0
 sphinxcontrib-programoutput==0.16
-msprime==0.7.4
 attrs==20.2.0
+-e git://github.com/tskit-dev/msprime.git@1cd73ef3f498903e022ebb0b2f6cfb9fa9564ab0#egg=msprime
```

Comment:
Similarly, I think this needs to be `git+https://`. And is `-e` really needed here?